gammapy/time/lightcurve.py

```diff
@@ -320,6 +321,136 @@ def create_fixed_time_bin(time_step, spectrum_extraction):
             ])
         return intervals
 
+    def create_fixed_significance_bin_lc_table(self, significance, significance_method, energy_range,
```

Comment:
The name and API of the methods to make time intervals should be consistent.
How about this?

* `create_fixed_time_bin` -> `make_time_intervals_fixes`
* `create_fixed_significance_bin_lc_table` -> `make_time_intervals_min_significance`

And change the existing method to return a table with start and stop time in the same format that we use here.
I.e. after one has computed the time interval, the rest of the code would be the same for users.